csharp/test/Ice/adapterDeactivation/Client.cs

```diff
@@ -8,7 +8,7 @@ public class Client : TestHelper
 {
     public override async Task runAsync(string[] args)
     {
-        using var communicator = initialize(ref args);
+        await using var communicator = initialize(ref args);
```

Comment:
I updated all (or at least most) of the tests with a runAsync to DisposeAsync the communicator.